test/test_tensor_creation_ops.py

```diff
@@ -543,10 +543,11 @@ def test_cat_out(self, device):
             self.assertEqual(y, expected_y)
             self.assertEqual(x, expected_x)
 
-    @dtypes(*all_types_and_complex())
+    @dtypes(*all_types_and_complex(), torch.uint16, torch.uint32, torch.uint64)
     def test_cat_out_fast_path_dim0_dim1(self, device, dtype):
+        int_types = integral_types_and(torch.uint16, torch.uint32, torch.uint64)
```

Comment:
nit: add a "with_barebones=False) arg to integral_types() ?

Reply:
Meh lol. Not sure this is buying all that much haha

Reply:
Yeah sounds fine if we don't need it in too many other places.